translations/fr/get5.phrases.txt

```diff
@@ -296,4 +408,64 @@
     {
         "fr"        "Le veto commencera dans {1} secondes."
     }
+    "SurrenderCommandNotEnabled"
+    {
+        "fr"        "La commande de capitulation n'est pas acctivée."
+    }
+    "SurrenderMinimumRoundDeficit"
+    {
+        "fr"        "Vous devez avoir terminé {1} manche(s) pour pouvoir capituler."
+    }
+    "SurrenderInitiated"
+    {
+        "fr"        "A vote to surrender was initiated by {1}. Your team must reach {2} votes within {3} seconds."
+    }
+    "SurrenderVoteStatus"
+    {
+        "fr"        "{1} des {2} votes de capitulation requis ont été exprimés."
+    }
+    "SurrenderSuccessful"
+    {
+        "fr"        "{1} a capitulé."
+    }
+    "SurrenderVoteFailed"
+    {
+        "fr"        "Trop peu de joueurs de votre équipe ont voté pour capituler."
+    }
+    "SurrenderOnCooldown"
+    {
+        "fr"        "Vous devez attendre {1} pour lancer un nouveau vote de capitulation."
+    }
+    "WinByForfeitAvailable"
+    {
+        "fr"        "{1} a quitté le serveur. {2} peut désormais entrer {3} pour initialiser un compte à rebours pour gagner à condition que votre équipe reste pleine."
+    }
+    "WinByForfeitRequiresFullTeam"
+    {
+        "fr"        "Votre équipe doit être complète pour pouvoir demander ou annuler une victoire par forfait."
```

Comment:
Votre équipe doit être complète pour pouvoir demander ou annuler une victoire par abandons.

Reply:
Currently says `Votre équipe doit être complète pour pouvoir demander ou annuler une victoire par abandon.`, again with no S.

Reply:
Both could work I think but with the S it would mean that each player of the team resign and without it it would mean that it is the team that's resigning